runing/services.py

```
@@ -0,0 +1,41 @@
+from typing import List, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)